src/main/xerus/monstercat/api/Playlist.kt

```diff
@@ -40,14 +44,25 @@ object Playlist {
 		else -> getNextTrack()
 	}
 	
+	private fun showUnsafeAlert(track: Track? = null) {
```

Comment:
please don't call it `safe`, stay with `licensable`

Reply:
b04be276b939a88481db98723de38f1fd5519247